fast_barcode_scanner/lib/src/overlays/rect_of_interest/rect_of_interest.dart

```diff
@@ -0,0 +1,129 @@
+import 'dart:ui';
+
+import 'package:fast_barcode_scanner_platform_interface/fast_barcode_scanner_platform_interface.dart';
+
+import '../../corner_point_utils.dart';
+
+typedef CodeFilter = bool Function(Barcode code);
+
+/// A simple description of a rect of interest. The Rect will be located in the
+/// center of the screen and will fill the width minus horizontal padding.
+abstract class RectOfInterest {
+  const RectOfInterest();
+
+  Rect rect(Rect previewWidgetRect);
+
+  CodeFilter codeFilter({
```

Comment:
I cannot check currently, if this method is exposed to the user, but it should be called something like `_buildCodeFilter` as it's a curried function aswell.

Reply:
It is used by `MaterialFinderOverlay`. Great naming suggestion. Will fix.